### DISCUSSION TITLE
CRUD: Change table action / inline Actions

### DISCUSSION THREAD

**Discussion by bernhardh** [orchidsoftware/platform#1717] [Q&A]
I am totally new to orchid, coming from laravel nova and just evaluating orchid at the moment. I really like what I saw so far, but currently I have two questions on the table view:

There are currently two actions available: View and Edit.

- Is there a way, to disable the "View" Action? In some cases, I dont wanna provide the "View" (legend?) at all.
- Is there a way to add actions to the action column? In laravel nova this feature is called inline actions. At least I am missing an delete action, but of course any custom action would be interesting (export, copy, move, etc.).

**Comment by tabuna** [comment]
> Is there a way, to disable the "View" Action? In some cases, I dont wanna provide the "View" (legend?) at all.

This is controlled by [policies](https://github.com/orchidsoftware/crud#policies). A very simple example:

```php
namespace App\Policies;

use App\Models\Post;
use App\Models\User;

class PostPolicy
{
    public function view(User $user, Post $post)
    {
        return false;
    }
}
```


> Is there a way to add actions to the action column? In laravel nova this feature is called inline actions. At least I am missing an delete action, but of course any custom action would be interesting (export, copy, move, etc.).

Not in the current version. The only way to add new actions to the top of the page (select from the list). This is described here: https://github.com/orchidsoftware/crud#actions

**Reply by wsdnathan** [reply]
Is there a simpler way to disable the "View" Action? If you have 10 Resources why must you create 10 complex Policies in order to do this? The View page just displays a blank page by default and otherwise is unused for me.

**Reply by tabuna** [reply]
You can use the same policy for different models, for example:

```php
class AuthServiceProvider extends ServiceProvider
{

    /**
     * The policy mappings for the application.
     *
     * @var array
     */
    protected $policies = [
        Post::class => DisableViewPolicy::class,
        Other::class => DisableViewPolicy::class,
    ];

   //...
}
```
